docs/samples/Microsoft.ML.Samples/Dynamic/ImageAnalytics/DnnFeaturizeImage.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using Microsoft.ML.Data;
+using Microsoft.ML.Transforms;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public static class DnnFeaturizeImage
+    {
+        public static void Example()
+        {
+            var mlContext = new MLContext();
+
+            // Downloading a few images, and an images.tsv file, which contains a list of the files from the dotnet/machinelearning/test/data/images/.
+            // If you inspect the fileSystem, after running this line, an "images" folder will be created, containing 4 images, and a .tsv file
+            // enumerating the images. 
+            var imagesDataFile = SamplesUtils.DatasetUtils.DownloadImages();
+
+            // Preview of the content of the images.tsv file, which lists the images to operate on
+            //
+            // imagePath    imageType
+            // tomato.bmp   tomato
+            // banana.jpg   banana
+            // hotdog.jpg   hotdog
+            // tomato.jpg   tomato
+
+            var data = mlContext.Data.CreateTextLoader(new TextLoader.Options()
+            {
+                Columns = new[]
+                {
+                        new TextLoader.Column("ImagePath", DataKind.String, 0),
+                        new TextLoader.Column("Name", DataKind.String, 1),
+                }
+            }).Load(imagesDataFile);
+
+            // Installing the Microsoft.ML.DNNImageFeaturizer packages copies the models in the
+            // `DnnImageModels` folder. The location of this folder is needed by the ResNet18 model.
+            string executablePath = typeof(DnnFeaturizeImage).Assembly.CodeBase;
+            string absolutePath = new Uri(executablePath).AbsolutePath;
+            var modelsParentFolder = Path.Combine(Directory.GetParent(absolutePath).FullName, "DnnImageModels");
```

Comment:
wow, does the user really have to do this? Can we have an API for it instead?

Reply:
it is due to bug: [https://github.com/dotnet/machinelearning/issues/2785](https://github.com/dotnet/machinelearning/issues/2785)

Once that is fixed, the users should not have to specify the folder, if they have already selected a model. 

i think we should leave it as is, until the bug gets fixed, than just remove the model path parameter. 

---
In reply to: [261430746](https://github.com/dotnet/machinelearning/pull/2799#discussion_r261430746) [](ancestors = 261430746)

Reply:
https://github.com/dotnet/machinelearning/pull/2809 this should fix this.
I build the packages from the repo and consumed these packages as a local nuget feed in a netcoreapp2.1 console project

Reply:
I'll wait for that PR to get in than, edit this and merge it. 

---
In reply to: [261456118](https://github.com/dotnet/machinelearning/pull/2799#discussion_r261456118) [](ancestors = 261456118)